tests/acceptance/features/apiSpaces/favorite.feature

```diff
@@ -0,0 +1,58 @@
+@api @files_sharing-app-required
```

Comment:
```suggestion
@api @skipOnOcV10
```
I don't think we need `@files_sharing-app-required` here, we have `@skipOnOcV10` in all of other feature file so maybe we can add it here too